articles/DeployExistingDjangoProject.md

```diff
@@ -83,10 +104,26 @@ create your virtualenv).
 ### Enter your virtualenv name
 
 Once that's done, **enter the name of your virtualenv** in the Virtualenv
-section on the web tab and click OK.  You can just use its short name
+section on the web tab and click OK.
+
+![virtualenv_web_app](path_to_the_screenshot)
+
+You can just use its short name
 "mysite-virtualenv", and it will automatically complete to its full path in
 /home/username/.virtualenvs.
 
+### Enter your code name
```

Comment:
this is really optional at the moment.  i would either remove this section altogether (to keep things as simple as possible).   if you really, really, really want to keep it, at least change the section title to: Optional: Enter path to source code